compiler/haskell-ide-core/src/Development/IDE/UtilGHC.hs

```diff
@@ -28,6 +28,7 @@ import qualified Packages
 import           Platform
 import qualified EnumSet
 import           Data.IORef
+import System.FilePath ((</>))
```

Comment:
```suggestion
import System.FilePath
```

I'd just import unqualified unless there is a good reason not to

Reply:
fixed.